compiler/rustc_interface/src/interface.rs

```diff
@@ -59,9 +60,25 @@ impl Compiler {
     }
 }
 
+fn registry_setup() {
```

Comment:
Could you explain what this function does? And in which case it should be called?

Reply:
I've removed `WorkerLocal` from `AttrIdGenerator` so this function can be removed.